src/pages/PositionDetails/Contents.tsx

```diff
@@ -341,20 +333,14 @@ export const Contents = (props: Props) => {
     const menu = [
       {
         disabled: !userHasBalance || !isConnected,
-        href: `/redeem`,
-        onClick: () => {
-          setValue(positionId)
-        },
+        href: `/redeem/${positionId}`,
         text: 'Redeem',
       },
-      {
-        disabled: !userHasBalance || !isConnected,
-        href: `/split`,
-        onClick: () => {
-          setValue(positionId)
-        },
-        text: 'Split',
-      },
+      // {
+      //   disabled: !userHasBalance || !isConnected,
+      //   href: `/split/${positionId}`,
+      //   text: 'Split',
+      // },
```

Comment:
Why this is commented ?

Reply:
Parameters are passed by URL, so for the split, we should choose to interpret the id as a condition or as a position but not both. These lines could be deleted, though.
Also, we can use another format but that wasn't discussed when the thread about this issue came up.